test_tar_changes.py

```diff
@@ -41,13 +42,15 @@ def distgitdir(tmpdir):
 
 
 def test_commit_distgit_amend(distgitdir):
-    suffix = 'GitLab-User: developer@example.com'
+    # Test if commit_distgit_amend(suffix) returns the correct formatted git commit msg
+    # suffix = tarchanges.check_gitlabuser()
+    suffix = "GitLab-User: developer@example.com"
```

Comment:
When the code is commented out like this on its own line, it's difficult to tell at a glance if it was a mistake.

What if we moved the comment onto the line it's describing, like this:

```suggestion
    suffix = "GitLab-User: developer@example.com"  # eg. from check_gitlabuser()
```